qgreenland/config/datasets/geus_mineral_occurrences.py

```diff
@@ -30,8 +30,8 @@
                 """Thorning, L., Christensen, L. A., Dawes, P. R., Garde, A. A.,
                 Heijboer, T. C., Kalvig, P., Larsen, L. M., Larsen, U., Nielsen, T. F.,
                 Rehnström, E. F., Thomassen, B., Thrane, K., Schjøth, F. & Secher, K. "Greenland
-                mineral occurrences map (GMOM)", GEUS GeoNetwork catalogue. Accessed on
```

Comment:
Is there a reason for removing the "Accessed on" bit? This makes it a little less ambiguous. E.g., the GrIMP suggested citation has "Date Accessed" before the date.

Reply:
@trey-stafford check out the change to the `util/metadata.py` module. My reasoning is that the text around access date was very inconsistent. Sometimes there was no text to tell readers what that date was. Now, `{{date_accessed}}` no longer returns only a date string, but the whole shebang, e.g.: `[Accessed on: 2023-08-18]`.

Reply:
Ah, I missed that! Thanks you :sweat_smile: